suites/quincy/nvmeof/tier-1_nvmeof_functl_Regression.yaml

```diff
@@ -123,7 +126,46 @@ tests:
       name: Test to perform Data Integrity test over NVMe-OF targets
       polarion-id:
 
-  # Cleanup Test Case
+# Run IOs using multiple-initiators against multi-subsystems based NVMe-OF targets using fio tool - Multi gateways in the cluster
+  - test:
+      abort-on-fail: true
+      config:
+        gw_node: node8                        # This covers multiple NVMEoF Gateway nodes in the cluster
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        install: true                           # Run SPDK with all pre-requisites
+        subsystems:                             # Configure subsystems with all sub-entities
+          - nqn: nqn.2016-06.io.spdk:cnode3
```

Comment:
As this is a new and second gateway, will we face any issue if we create sub_nqn  again with nqn.2016-06.io.spdk:cnode1 and nqn.2016-06.io.spdk:cnode2 is created instead of nqn.2016-06.io.spdk:cnode3 and nqn.2016-06.io.spdk:cnode4 ?

Reply:
No should not create any issues since its a different gateway node- I gave different names for testing purpose. I can check and let you know for confirmation